packages/analytics-types/src/autocapture.ts

```diff
@@ -0,0 +1,78 @@
+import { Logger } from './logger';
+
+export type ActionType = 'click' | 'change';
+
+/**
+ * Default CSS selectors to define which elements on the page to track.
+ * Extend this list to include additional elements to track. For example:
+ * ```
+ * autocapturePlugin({
+ *    cssSelectorAllowlist: [...DEFAULT_CSS_SELECTOR_ALLOWLIST, ".my-class"],
+ * })
+ * ```
+ */
+export const DEFAULT_CSS_SELECTOR_ALLOWLIST = [
+  'a',
+  'button',
+  'input',
+  'select',
+  'textarea',
+  'label',
+  '[data-amp-default-track]',
+  '.amp-default-track',
+];
+
+/**
+ * Default prefix to allo the plugin to capture data attributes as an event property.
+ */
+export const DEFAULT_DATA_ATTRIBUTE_PREFIX = 'data-amp-track-';
+
+export interface AutocaptureOptions {
```

Comment:
Did you update the Autcapture plugin to use the core SDK types (instead of refining). Just want to check this is defined in a single location going forward.

Reply:
I will do that later as the description says
 
> Will update @amplitude/plugin-autocapture-browser to use AutocaptureOptions in @amplitude/analytics-types later in [AMP-102833](https://amplitude.atlassian.net/browse/AMP-102833?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ)

Will do it when this feature branch is merged into the main branch and publish the package without a feature tag.